pkg/instance/handler.go

```diff
@@ -43,16 +43,19 @@ type userClientHandler interface {
 	FindUserById(token string, id uint) (*models.User, error)
 }
 
-type CreateInstanceRequest struct {
-	Name      string `json:"name" binding:"required,dns_rfc1035_label"`
-	GroupName string `json:"groupName" binding:"required"`
-	StackName string `json:"stackName" binding:"required"`
+type LaunchInstanceRequest struct {
+	Name               string                            `json:"name" binding:"required,dns_rfc1035_label"`
+	Group              string                            `json:"groupName" binding:"required"`
+	Stack              string                            `json:"stackName" binding:"required"`
+	RequiredParameters []model.InstanceRequiredParameter `json:"requiredParameters"`
```

Comment:
I would not use our model in the user facing API. I mean in `enrichParameters` you do override `InstanceID`, `StackName` but this opens up a spot that can either introduce security issues or bugs.